src/replication/replicator.rst

```diff
@@ -43,107 +48,151 @@ Let's say you POST the following document into ``_replicator``:
 
     {
         "_id": "my_rep",
-        "source":  "http://myserver.com:5984/foo",
+        "source":  "http://user:pass@localhost:5984/foo",
```

Comment:
Perhaps a different hostname here, just to make sure it's still clear that replication is not a "private to me" concept, but a "works between machines" concept?